packages/commerce-sdk-react/src/auth/storage/cookie.test.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import Cookies from 'js-cookie'
+import {CookieStorage} from './cookie'
+
+jest.mock('js-cookie', () => ({
+    set: jest.fn(),
+    get: jest.fn(),
+    remove: jest.fn()
+}))
+
+describe('CookieStorage', () => {
+    let storage: CookieStorage
+
+    beforeEach(() => {
+        storage = new CookieStorage()
+        jest.useFakeTimers()
+        Date.now = jest.fn(() => new Date('2024-09-03T21:00:00Z').getTime())
```

Comment:
prototype pollution! `Date.now` is changed for the entire node process for the jest tests. I think ya may need to revert the implementation of Date.now after these tests are done